subgraph/config/arbitrum.json

```diff
@@ -0,0 +1,7 @@
+{
+    "network": "arbitrum-one",
+    "unlock" : "0x1FF7e338d5E582138C46044dc238543Ce555C963",
+    "unlockStartBlock": "17429533",
+    "unlock2" : "0x0000000000000000000000000000000000000000",
+    "unlockStartBlock2": "17429533"
+  }
```

Comment:
we have all these info into the `@unlock-protocol/networks` package so it may be easier to just generate the networks files at build step?

Reply:
Could definitely do this. Would you like me to remove the unused functions in `subgraph/bin` as well ?

Reply:
I was going to ultimately suggest to split out the subgraph from the monorepo as setting up the gitflow & deploy on commit might be trickier.

Reply:
Have added it to the network build step. Configs generated from package as requested

Reply:
@clemsos Let me know if this is sufficient. I have added networkName to subgraph object for those that did not have it.

Reply:
yes seems fine, thanks